<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="windowTitle">
   <string>Create New Project</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">

   <!-- Form Layout for all fields except Notes -->
   <item>
    <layout class="QFormLayout" name="formLayout">

     <item row="0" column="0">
      <widget class="QLabel" name="labelProjectName">
       <property name="text">
        <string>Project Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="projectNameEdit"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="labelLocation">
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="locationLayout">
       <item>
        <widget class="QLineEdit" name="locationEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="labelFPS">
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="fpsComboBox">
       <item><property name="text"><string>12</string></property></item>
       <item><property name="text"><string>24</string></property></item>
       <item><property name="text"><string>25</string></property></item>
       <item><property name="text"><string>30</string></property></item>
       <item><property name="text"><string>60</string></property></item>
      </widget>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="labelResolution">
       <property name="text">
        <string>Resolution Format</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="resolutionComboBox">
       <item><property name="text"><string>1280x720</string></property></item>
       <item><property name="text"><string>1920x1080</string></property></item>
       <item><property name="text"><string>3840x2160</string></property></item>
      </widget>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="labelSafeFrame">
       <property name="text">
        <string>Safe Frame</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="safeFrameComboBox">
       <item><property name="text"><string>80%</string></property></item>
       <item><property name="text"><string>90%</string></property></item>
       <item><property name="text"><string>93%</string></property></item>
       <item><property name="text"><string>Custom</string></property></item>
      </widget>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="labelEstimatedDuration">
       <property name="text">
        <string>Estimated Duration (min)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="estimatedDurationEdit"/>
     </item>

     <item row="6" column="0">
      <widget class="QLabel" name="labelAspectRatio">
       <property name="text">
        <string>Aspect Ratio</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="aspectRatioComboBox">
       <item><property name="text"><string>16:9</string></property></item>
       <item><property name="text"><string>2.35:1</string></property></item>
       <item><property name="text"><string>4:3</string></property></item>
      </widget>
     </item>

     <item row="7" column="0">
      <widget class="QLabel" name="labelProjectCode">
       <property name="text">
        <string>Project Code</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="projectCodeEdit"/>
     </item>

     <item row="8" column="0">
      <widget class="QLabel" name="labelDirector">
       <property name="text">
        <string>Director / Creator</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="directorEdit"/>
     </item>

    </layout>
   </item>

   <!-- Notes Section -->
   <item>
    <widget class="QLabel" name="labelNotes">
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="notesEdit"/>
   </item>

   <!-- Buttons -->
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>